flows/src/main/java/com/softwaremill/jox/flows/Flow.java

```diff
@@ -1678,6 +1676,10 @@ public void runToFile(Path path) throws Exception {
                     }
                 });
                 close(channel, null);
+            } catch (JoxScopeExecutionException e) {
+                Exception cause = (Exception) e.getCause();
+                close(channel, cause);
```

Comment:
maybe I merged too fast ;) shouldn't we not-unwrap here as well?

Reply:
Hmm, I might have missed that bit, let me open another PR to revert it as well

Reply:
Fixed: https://github.com/softwaremill/jox/pull/89